components/rdv/EditionRdvForm.tsx

```diff
@@ -233,6 +235,15 @@ export function EditionRdvForm({
     else onChanges(false)
   })
 
+  function emailInvitationText(conseillerIsCreator: boolean) {
+    if (conseillerIsCreator) {
+      return `Intégrer ce rendez-vous à mon agenda via l’adresse mail suivante :
+      ${conseillerEmail}`
+    } else {
+      return "Le créateur du rendez-vous recevra un mail pour l'informer de la modification."
```

Comment:
```suggestion
      return "Le créateur du rendez-vous recevra un e-mail pour l'informer de la modification."
```
idem https://zeroheight.com/2f9442f46/p/36c04e-lexique/b/75adeb, le mentionner sur le ticket ou aux PO au cas où :-)